packages/@aws-cdk/aws-ecs/lib/fargate/fargate-service.ts

```diff
@@ -65,6 +67,30 @@ export interface IFargateService extends IService {
 
 }
 
+/**
+ * The properties to import from the service using the Fargate launch type.
+ */
+export interface FargateServiceAttributes {
+  /**
+   * The cluster that hosts the service.
+   */
+  readonly cluster: ICluster;
+
+  /**
+   * The service ARN.
+   *
+   * @default - generated from serviceName
+   */
+  readonly serviceArn?: string;
+
+  /**
+   * The name of the service.
+   *
+   * @default - generated from serviceArn
```

Comment:
This comment is wrong (should be "either this, or {@link serviceArn}, is required").